src/plugins/workspace/public/components/workspace_creator/workspace_creator.tsx

```diff
@@ -78,7 +78,7 @@ export const WorkspaceCreator = () => {
               onSubmit={handleWorkspaceFormSubmit}
               opType={WORKSPACE_OP_TYPE_CREATE}
               permissionEnabled={isPermissionEnabled}
-              permissionFirstUserDeletable
+              permissionLastAdminItemDeletable
```

Comment:
According to our last discussion, it seems that last admin item will always be nondeletable, no matter in create/update, so do we need this flag anymore?

Reply:
Upon workspace creation, we will implicitly grant current user admin permission, which ensures that every workspace has at least an admin user. Do you think it is okay to delete the last admin for the create page?

Reply:
Oh, my default, forget it.